src/test/java/com/faforever/client/chat/AbstractChatTabControllerTest.java

```diff
@@ -322,4 +323,11 @@ public void getMessageCssClassChatOnlyNullPlayerInfoBean() {
     String playerName = "somePlayer";
     assertEquals(instance.getMessageCssClass(playerName), CSS_CLASS_CHAT_ONLY);
   }
+
+  @Test
+  public void testChannelNamesTransformedToHyperlinks() {
+    String output = instance.replaceChannelNamesWithHyperlinks("Go to #moderation and report a user");
+    assertThat(output, containsString("#moderation"));
+    assertThat(output, containsString("<a"));
```

Comment:
what about just writing the expected string down here?

Reply:
just thought it would be more open towards improvements but i guess you are right there